website/src/pages/Blog/Author/index.tsx

```diff
@@ -45,17 +45,17 @@ const Blog: React.FC = () => {
   const classes = useStyles();
   const { currentOrigin } = useCurrentHost();
   const [ authorMetadata, setAuthorMetadata ] = useState<AuthorMetadata | null>(null);
-  const [jsonMdData, setJsonMdData] = useState<blog[]>([
-    { title: "",
-      author: "",
-      excerpt: "",
-      author_info: "",
-      date: "",
-      tags: [""],
-      content: "",
-      id: 0,
-      slug: "" }
-  ]);
+  const [authorArticlesData, setAuthorArticlesData] = useState<blog[] | null>([
```

Comment:
```suggestion
  const [authorBlogsData, setAuthorBlogData] = useState<blog[] | null>([
```

For consistent naming